frontend/widgets/OBSBasic.hpp

```diff
@@ -26,6 +26,7 @@
 #include <utility/OBSCanvas.hpp>
 #include <utility/VCamConfig.hpp>
 #include <utility/platform.hpp>
+#include <utility/PreviewProgramSizeObserver.hpp>
```

Comment:
Technically, uppercase letters come before lowercase letters, though I think there are places we're inconsistent with this.

Reply:
I usually just tell an application to sort these case-sensitive and call it a day.

So the order (consistent with the results of the frontend refactor) would be:

```C++
#include <utility/OBSCanvas.hpp>
#include <utility/PreviewProgramSizeObserver.hpp>
#include <utility/VCamConfig.hpp>
#include <utility/platform.hpp>
#include <utility/undo_stack.hpp>
```